com.unity.ml-agents/Runtime/Sensors/IDimensionPropertiesSensor.cs

```diff
@@ -0,0 +1,47 @@
+namespace Unity.MLAgents.Sensors
+{
+
+    /// <summary>
+    /// The Dimension property flags of the observations
+    /// </summary>
+    [System.Flags]
+    public enum DimensionProperty
+    {
+        /// <summary>
+        ///     No properties specified.
+        /// </summary>
+        Unspecified = 0,
+
+        /// <summary>
+        /// No Property of the observation in that dimension. Observation can be processed with
+        /// Fully connected networks.
+        /// </summary>
+        None = 1,
+
+        /// <summary>
+        /// Means it is possible to do a convolution in this dimension.
```

Comment:
I wonder if it will be better by saying something like "suitable/meaningful to apply a convolution".

(It's possible to do a conv anyways.)